<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ingresar</class>
 <widget class="QDialog" name="Ingresar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>173</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>340</width>
    <height>173</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>340</width>
    <height>173</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LNMusic - Ingresar</string>
  </property>
  <widget class="QPushButton" name="btnIngresar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ingresar</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>img/Knob Forward.png</normaloff>img/Knob Forward.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNuevo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>98</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Nuevo</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>img/Knob Add.png</normaloff>img/Knob Add.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="lblUsuario">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>19</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Usuario:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbUsuario">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>19</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtContra">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>69</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>**********</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblContra">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Contrase√±a:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
